README.md

```diff
@@ -68,23 +68,38 @@ zuul:
     enabled: true 
     repository: REDIS 
     behind-proxy: true
-    default-policy: #optional - will apply unless specific policy exists
-      limit: 10 #optional - request number limit per refresh interval window
-      quota: 1000 #optional - request time limit per refresh interval window (in seconds)
-      refresh-interval: 60 #default value (in seconds)
-      type: #optional
-        - user
-        - origin
-        - url
     policies:
-      myServiceId:
-        limit: 10 #optional - request number limit per refresh interval window
-        quota: 1000 #optional - request time limit per refresh interval window (in seconds)
-        refresh-interval: 60 #default value (in seconds)
-        type: #optional
-          - user
-          - origin
-          - url
+      #optional - will apply unless specific policy exists
+      #optional name - this policy name , types value is not empty recommend use
+      - name: all
+        #optional - request number limit per refresh interval window
+        limit: 50
+        #optional - request time limit per refresh interval window (in seconds)
+        quota: 50000
+        #default value (in seconds)
+        refresh-interval: 1
+        #optional - can select request and mark key 
+        types:
```

Comment:
is the url the only one with value? I'm not sure we want it to be a map, also policies by url is not consistent with zuul aproach

Reply:
Then we want to support the rate limit based on url should be how to do?